src/bin/casr-cluster.rs

```diff
@@ -408,10 +408,77 @@ fn dedup(indir: &Path, outdir: Option<PathBuf>) -> Result<(usize, usize)> {
     Ok((before, after))
 }
 
+/// Merge sub directory to main
+///
+/// # Arguments
+///
+/// * `new` - path to directory with new CASR reports
+///
+/// * `storage` - path to storage directory with CASR reports
```

Comment:
Maybe input, output? the same as in help message

Reply:
done